Tensile/KernelWriter.py

```diff
@@ -1220,9 +1244,25 @@ def makeSubIterSchedule(self, kernel, localReadCode, iteration, pointerLWCode, p
           barrier.addInst("s_barrier","")
           iterCode.addCode(barrier)
 
-        if kernel["StorePriorityOpt"] and kernel["PrefetchGlobalRead"] == 2 and \
-            (mfmaIndex == self.lwStartMfmaIndex or mfmaIndex == self.barrierMfmaIndex+2) :
-          iterCode.addInst("s_setprio 3","store optimization")
+        if kernel["StorePriorityOpt"]:
```

Comment:
Do the changes to priority impact other data types?

Reply:
Actually, yes.
I asked the person who implemented this logic first and he considered only DGEMM (even though this is applicable for other types).